docs/samples/Microsoft.ML.Samples/Dynamic/Calibrator.cs

```diff
@@ -1,4 +1,8 @@
-﻿using System;
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
I don't think we put this header to samples

Reply:
No idea why it put my reply here when I was replying to your other thing...